src/Silex/Provider/RoutingServiceProvider.php

```diff
@@ -30,13 +29,20 @@
  */
 class RoutingServiceProvider implements ServiceProviderInterface
 {
+    /**
+     * Added for BC reason only.
+     */
+    protected const BC_FEATURE_FLAG_LOCALE_LISTENER = 'BC_FEATURE_FLAG_LOCALE_LISTENER';
+
     /**
      * @param \Silex\Application $app
      *
      * @return void
      */
     public function register(Application $app)
     {
+        $app[static::BC_FEATURE_FLAG_LOCALE_LISTENER] = true;
```

Comment:
why hardcoded `true` value assignment here?

Reply:
This enables us to disable the internal Silex code when we add our own code. In our own code those `BC_FEATURE_FLAG_*` are then set to false.